fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/ClientLoanIntegrationTest.java

```diff
@@ -65,7 +65,7 @@
  * Client Loan Integration Test for checking Loan Application Repayments Schedule, loan charges, penalties, loan
  * repayments and verifying accounting transactions
  */
-@SuppressWarnings({ "rawtypes", "unchecked" })
+@SuppressWarnings({ "rawtypes", "unchecked", "AbbreviationAsWordInName" })
 public class ClientLoanIntegrationTest {
```

Comment:
What was the reason for suppressing for this file?

Reply:
Only this remains, I suppressed it here since there were 130-140 letter big function names and many of them, I thought it was really easy to read them in the way they are now. 

Also, I found those changes unnecessary since other errors in that file were solved already. test functions name should not matter much?

What do you think?

Reply:
@thesmallstar I would actually have to agree with @ptuomola that if just a little bit more work, it would be nice to finish this up.. but it depends on the effort, of course. How about we agree that we don't use `@SuppressWarnings("AbbreviationAsWordInName")` at least in the 2-3 `main` (non-`test`) code places I pointed out in this review, but that if there is any `test` code where it's an unreasonable amount of work to change it, we merge it anyway - would that seem fair?